packages/syft/src/syft/node/node.py

```diff
@@ -1608,6 +1609,10 @@ def create_default_worker_pool(node: Node) -> SyftError | None:
             image_uid=default_image.id,
             num_workers=worker_count,
         )
+    elif isinstance(default_worker_pool, SyftError):
```

Comment:
Lets move this check before the `create_default_image` method itself (Move to Line 1567)